katago_server/trainings/models/network.py

```diff
@@ -41,11 +67,16 @@ class Meta:
         verbose_name = _("Network")
         verbose_name_plural = _("Networks")
 
+    # QUESTION (lightvector): Do any changes need to be made such that this can be a different identifier supplied by the training process?
     uuid = UUIDField(_("unique identifier"), default=uuid.uuid4, db_index=True)
     created_at = DateTimeField(_("creation date"), auto_now_add=True)
+    # QUESTION (lightvector): What is parent_network for?
     parent_network = ForeignKey("self", null=True, blank=True, related_name="variants", on_delete=PROTECT)
```

Comment:
We discussed that a while ago. We need parent network for bayesian prior of bayesian elo. 

Right now the parent is always network N-1 but in case of official and community run we have

![image](https://user-images.githubusercontent.com/13785185/79079566-3deb4900-7d10-11ea-96c2-864f148051b4.png)

Reply:
Ah, right. Okay. :)